sql/hive/src/main/scala/org/apache/spark/sql/hive/client/ClientWrapper.scala

```diff
@@ -166,7 +168,11 @@ private[hive] class ClientWrapper(
           " specified in spark.yarn.keytab does not exist")
       } else {
         logInfo("Attempting to login to Kerberos" +
-          s" using principal: ${principalName} and keytab: ${keytabFileName}")
+        s" using principal: ${principalName} and keytab: ${keytabFileName}")
+        val hadoopConfiguration = new Configuration()
+        hadoopConfiguration.set(CommonConfigurationKeysPublic.HADOOP_SECURITY_AUTHENTICATION,
```

Comment:
This seems wrong. If you are talking to hadoop cluster that is secure, your client configuration should already have this. Setting this here alone is not going to help, since all other instances used all over Spark will not have this information (where required).